pylint/testutils/decorator.py

```diff
@@ -13,6 +15,42 @@ def set_config(**kwargs):
     allows this decorator to be used on parametrized test cases.
     """
 
+    def _wrapper(fun):
```

Comment:
```suggestion
    def _wrapper(fun: typing.Callable) -> typing.Callable:
```
Maybe, it's overkill and we don't need typing for inner function ?

Reply:
Think we might want to hold off on this when we type `testutils` completely. Adding a general `Callable` doesn't add too much info right now imo.